bluebrain/repo-bluebrain/packages/nmodl/package.py

```diff
@@ -7,85 +7,110 @@
 
 
 class Nmodl(CMakePackage):
-    """Code Generation Framework For NEURON MODeling Language """
+    """Code Generation Framework For NEURON MODeling Language"""
 
     homepage = "https://github.com/BlueBrain/nmodl.git"
-    url      = "https://github.com/BlueBrain/nmodl.git"
-    git      = "https://github.com/BlueBrain/nmodl.git"
+    url = "https://github.com/BlueBrain/nmodl.git"
+    git = "https://github.com/BlueBrain/nmodl.git"
 
     # 0.3.1 > 0.3.0.20220110 > 0.3.0 > 0.3b > 0.3 to Spack
-    version('develop', branch='master', submodules=True)
-    version('llvm', branch='llvm', submodules=True)
+    version("develop", branch="master", submodules=True)
+    version("llvm", branch="llvm", submodules=True)
+    # This is the merge commit of #875, which allows catch2 etc. to be dependencies
+    version("0.3.0.20220531", commit="d63a061ee01b1fd6b14971644bb7fa3efeee20b0")
     # For deployment; nmodl@0.3.0%nvhpc@21.11 doesn't build with eigen/intrinsics errors
-    version('0.3.0.20220110', commit='9e0a6f260ac2e6fad068a39ea3bdf7aa7a6f4ee0')
-    version('0.3.0', tag='0.3')
-    version('0.3b', commit="c30ea06", submodules=True)
-    version('0.3a', commit="86fc52d", submodules=True)
-    version('0.2', tag='0.2', submodules=True)
+    version("0.3.0.20220110", commit="9e0a6f260ac2e6fad068a39ea3bdf7aa7a6f4ee0")
+    version("0.3.0", tag="0.3")
+    version("0.3b", commit="c30ea06", submodules=True)
+    version("0.3a", commit="86fc52d", submodules=True)
+    version("0.2", tag="0.2", submodules=True)
 
     variant("legacy-unit", default=True, description="Enable legacy units")
     variant("python", default=False, description="Enable python bindings")
     variant("llvm", default=False, description="Enable llvm codegen")
-    variant("llvm_cuda", default=False, description="Enable llvm codegen with CUDA backend")
+    variant(
+        "llvm_cuda", default=False, description="Enable llvm codegen with CUDA backend"
+    )
 
     # Build with `ninja` instead of `make`
-    generator = 'Ninja'
-    depends_on('ninja', type='build')
-    depends_on('llvm', when='+llvm')
-    depends_on('cuda', when='+llvm_cuda')
-
-    conflicts('+llvm', when='@0.2:0.3.0.20220110', msg='cannot enable LLVM backend outside of llvm version')
-    conflicts('+llvm_cuda', when='@0.2:0.3.0.20220110', msg='cannot enable CUDA LLVM backend outside of llvm version')
+    generator = "Ninja"
+    depends_on("ninja", type="build")
+    depends_on("llvm", when="+llvm")
+    depends_on("cuda", when="+llvm_cuda")
+
+    conflicts(
+        "+llvm",
+        when="@0.2:0.3.0.20220531,develop",
```

Comment:
@matz-e do you know a better way of writing "it conflicts with all versions except `@llvm`"?